src/main/webapp/components/Table.js

```diff
@@ -128,6 +139,31 @@ export const Table = hh(class Table extends Component {
     }, ["Info Link"])
   };
 
+  redirectToDocumentLink = (cell, row) => {
```

Comment:
nit: the name first led me to think this was redirecting the user to a document download link but it's really just a `documentLink` ... there's not redirecting happening here.